instructions.md

```diff
@@ -0,0 +1,234 @@
+
+#### Hint
+
+Think about what the main components of the application will be and how they will interact with each other. Think about how the state will look, which components will read the state, and which components will dispatch actions.
+
+### Create Files and Run it Locally
+
+On your computer, create the files needed for your React and Redux Toolkit application. Run your application locally to see what it looks like in the browser.
+
+#### Hint
+
+You can use [create-react-app](https://create-react-app.dev/) to start your React application. If you want to set up Redux automatically, you can use [the Redux flag](https://redux-toolkit.js.org/introduction/quick-start#using-create-react-app).
+
+### Version control
+
+If you used `create-react-app` above, you can skip this step. `create-react-app` automatically configures the local Git repository. You will still need to set up a remote origin though, you can learn more [here](TODO:REPLACE_WITH_BENS_ARTICLE).
+
+Set up the folder you created previously to be a Git repository. Push the initial files to a repository on GitHub. You should be consistently committing your changes throughout the project. Make sure to have meaningful commit messages.
+
+#### HINT
+
+To initialize your Git repository, you can run the below code in your terminal, where `application` is the name of your project folder.
+
+```git init application```
+
+If you want a refresher on the syntax, look back at the [Git cheat sheet](https://education.github.com/git-cheat-sheet-education.pdf).
+
+### Build the Components
+
+Start building your application with mock data. You should focus on how your components will interact with the data rather than how it will get the data from APIs at this point (that comes in a later task). Remember to build reusable components and to keep your components small.
+
+Things to keep in mind:
+
+- You are welcome to use any libraries to help you build features
+- You are welcome to use other APIs that are not listed above
+- You should be consistent on the folder structure. Remember to think about features, connected components, and presentational components.
+
+### Connect your Components to Redux
+
+Now that you've built your components, you will want to connect them to the Redux store. Related data should belong in the same slice and actions should describe events happening in your application. At this point your application is not connected with the APIs so feel free to set the initial states for the slices to mock data.
+
+HINT
+You'll need at least four slices: one for each API and one for the goals list.
+
+### Connect your Application with the APIs
+
+Connect your application with the APIs needed for your features. When interacting with APIs, don't forget to handle the loading and error states.
```

Comment:
I'm not sure if this is necessary anymore. I think we can get rid of this task, add a "Connect your Application with the..." to the title of the three API-related tasks, and mention "When interacting with APIs, don't forget to handle the loading and error states".